server/src/main/java/com/hedera/block/server/persistence/storage/read/BlockAsDirReaderBuilder.java

```diff
@@ -55,16 +58,17 @@ public static BlockAsDirReaderBuilder newBuilder(@NonNull PersistenceStorageConf
      * and directories.
      *
      * <p>By default, the block reader will use the permissions defined in {@link
-     * FileUtils#defaultPerms}. This method is primarily used for testing purposes. Default values
-     * should be sufficient for production use.
+     * com.hedera.block.common.utils.FileUtilities#DEFAULT_DIR_PERMISSIONS}. This method is
+     * primarily used for testing purposes. Default values should be sufficient for production use.
```

Comment:
Just remove the paragraph, and leave the default permissions as an internal detail of the commons module.

Reply:
:white_check_mark: Done.